test/massa-proto-cli.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { getProtoFunction, ProtoFile } from '../packages/massa-proto-cli/src/protobuf';
+
+test('test commands', () => {
+  // TODO: test commands
+  expect(true).toBe(true);
+});
+
+/** 
+ * extract all the necessary information from a proto file
+ * (uses getProtoFunction from packages/massa-proto-cli/src/protobuf.ts)
+ */
+test('test protobuf.ts: getProtoFunction', () => {
```

Comment:
That's not a unit test, unit test as for objective to test single feature so since each field returned are made with different code parts within the function it must have a test for each fields.